### PR TITLE
GRAS Recorders to Building orientation

### PR DESCRIPTION
**Pull request by salichon**
Changed sensor orientation to fit building walls loc 23/24

### CODE REVIEW

**Comment by CBurton90** [comment]
@salichon Applies to LC20 too.

**Comment by salichon** [comment]
- updated Loc 20

File: install/recorders.csv [hunk 1]
```diff
@@ -742,11 +742,14 @@ Canterbury Seismic Instruments,EQR120,EQR120,46617,HOCS,20,0,,0,0,2023-07-10T23:
 Canterbury Seismic Instruments,EQR120,EQR120,46618,MENS,20,0,,0,0,2023-06-30T03:08:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46619,JSCB,23,252,,0,-2.5,2024-02-29T02:57:00Z,9999-01-01T00:00:00Z
 Canterbury Seismic Instruments,EQR120,EQR120,46621,PTOS,20,187,,0,0,2023-02-01T01:21:00Z,9999-01-01T00:00:00Z
-Canterbury Seismic Instruments,EQR120,EQR120,46622,GRAS,20,0,,0,0,2023-07-03T23:29:00Z,9999-01-01T00:00:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46622,GRAS,20,0,,0,0,2023-07-03T23:29:00Z,2024-10-21T03:20:00Z
+Canterbury Seismic Instruments,EQR120,EQR120,46622,GRAS,20,50,0,0,2024-10-21T03:30:00Z,9999-01-01T00:00:00Z
```
Comment (Thomas-Benson) [inline install/recorders.csv:746]: Needs an extra comma here,  `GRAS,20,50,,0,0` instead of `GRAS,20,50,0,0`

**Review by Thomas-Benson — CHANGES_REQUESTED** [review]

File: install/recorders.csv (hunk 1 above)
Comment (salichon) [reply install/recorders.csv:746]: too quick .. thanks!

**Comment by salichon** [comment]
Heloo @Thomas-Benson  :) 
can you have a final eye :) thanks ! 
still  in search of the height - but so far so good.

**Review by Thomas-Benson — APPROVED** [review]
Orientation update looks good

**Pull request merged**